### PR TITLE
[Feature] Add AppView

### PR DESCRIPTION
**Pull request by dwojcik96**
Changes:
- Update package.json
- Update structure & styles of App.js
- Change app's name

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+              WhoBorrowedThat?
           </p>
-      </div>
+      </AppView>
     );
   }
 }
 export default App;
 
+const AppView = styled.div`
```
Comment (tkrajewski) [inline src/App.js]: AppView should be defined before it's being used mate 🍺 Also the good practice is to keep styled components in a separate file, but it's also fine to keep it in the components code.

File: src/App.js [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
+  background-color: #212121;
```
Comment (tkrajewski) [inline src/App.js]: Consider keeping colors, sizes etc. in one file named variables.js and just import what you need.

File: src/index.html [hunk 3]
```diff
@@ -3,17 +3,11 @@
 <head>
     <meta charset="utf-8">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" >
-    <title>Lang!</title>
+    <title>WhoBorrowedThat?</title>
```
Comment (tkrajewski) [inline src/index.html]: Yeah, exactly, who!? :D

File: src/App.js [hunk 4]
```diff
@@ -1,17 +1,23 @@
 import React, { Component } from "react";
 import ReactDOM from "react-dom";
+import styled from 'styled-components'
 class App extends Component {
```
Comment (tkrajewski) [inline src/App.js]: You can write this in a shorter way:
```
const App = () => ((
  <AppView>
    ...
  </AppView>
))
```

File: src/App.js [hunk 5]
```diff
@@ -1,17 +1,23 @@
 import React, { Component } from "react";
 import ReactDOM from "react-dom";
+import styled from 'styled-components'
```
Comment (tkrajewski) [inline src/App.js]: Missing spacing between imports and class definition. Missing semicolon.

File: src/App.js (hunk 1 above)
Comment (tkrajewski) [reply src/App.js]: The idea is to keep the good structure per component. We can discuss the project structure tomorrow and I will fully explain how to bite that.

**Comment by tkrajewski** [comment]
@dwojcik96 Also it is good to include change log in PR description : ) just to have things clean and easy to maintain.

File: src/App.js (hunk 1 above)
Comment (dwojcik96) [reply src/App.js]: @tkrajewski sounds good!

**Pull request merged**